### ISSUE TITLE
Release 2.1.0!

### ISSUE THREAD

**Issue by AHAAAAAAA** [AHAAAAAAA/PokemonGo-Map#1875]
We've paused accepting PRs while we clean up and test the develop branch to merge into master. I'm super sorry for the delays and hold up, but code quality is plummeting and app is getting bloated.

Currently, we're only testing, bug fixing, and working through [these PRs](https://github.com/AHAAAAAAA/PokemonGo-Map/pulls?q=is%3Aopen+is%3Apr+milestone%3A2.01)  as well address issue #1806. I'd appreciate all the help that we could get to work through those.

**Comment by megahambone** [comment]
@AHAAAAAAA - I really wish you'd consider PR #725 in your list above - I've updated it and merged it and rebased it a number of times (per request) - I've even found that the develop branch has a number of dup functions in it at lines 617 - 679 and at 894 - 957 (the latter seem to be more recent)
If not, could you let me know when the develop branch is finalized if I need to refactor?

**Comment by kevinpark1217** [comment]
@AHAAAAAAA What is up with leaving a lot of PR labeled as enhancement and not being merged. Will these PRs merged all at once for 2.01?

**Comment by AHAAAAAAA** [comment]
@megahambone You're correct and sorry for leaving this for so long, there's 60 PRs and it's hard to keep track of anything. There's a current location feature and it's broken so I'm considering yours a fix. There's #1816 PR to clean up map.js in this milestone.

**Comment by AHAAAAAAA** [comment]
@kevinpark1217 They just describe the purpose of the PR, and the one's not included in the 2.01 milestone will be merged only after that is complete.

**Comment by megahambone** [comment]
@AHAAAAAAA - Thank you! Ultimately, I was concerned that you'd do a major refactor of develop and that I'd be back at square one. I know that this is a very popular repo with a lot of spinning plates so I respect that it's got to be a giant pain to manage it all. I appreciate the response!

**Comment by ofiras** [comment]
@AHAAAAAAA I think @megahambone 's PR could make this repo extremely better, I am using it on android via ngrok and it works like a charm. Please consider making it a priority.

**Comment by FrostTheFox** [comment]
Can we readd example.py in this latest version?

**Comment by Sejadis** [comment]
very nice!

as 2.0.2 could potentially be a "add-a-lot of-features" update imo it would be nice if the external alerts would be priority - im still using one of the older mail alert pr's and seeing all those slack, pushbullet, and much more pr's i really wait to see them inplemented :)

**Comment by AHAAAAAAA** [comment]
@Sejadis I also would love those alerts. Definitely 2.02 milestone.

**Comment by Chlodochar** [comment]
The features are great, but this is the perfect time for those file-wide/project-wide fixes and code cleanups. Last couple of days I've seen features being added while there were still open bugs and code formatting issues in the files that the features were touching. File-wide bug fixes and cleanups need a rebase after each and every feature that was merged. (Every couple of hours it seemed :P)

File-wide code formatting can be done afterwards automatically (before the release). Project-wide bug fixes, structure and code cleanups cannot.

**Comment by mquandalle** [comment]
You could also configure ESLint and run on a free CI system (eg Travis CI) to enforce a consistent code style—and avoid potential errors—on PRs.

**Comment by AHAAAAAAA** [comment]
@Chlodochar I agree, this is why I also linked #1806 to the milestone. We need to clean up and have proper code quality control. I have travis already linked up, we just need to add the config file and tests.

**Comment by kintera** [comment]
@AHAAAAAAA I have finally gotten a few days in the code now I will attempt to help with the code cleanup as I can. Is it just a optimization and clean up we are looking for or would you rather me focus my time and effort on Tech documentation?

**Comment by AHAAAAAAA** [comment]
@kintera we're doing bug fixes and clean up across code base. Documentation fixes, formatting, and clean up also highly appreciated. I will open that up for edits.

**Comment by rneu31** [comment]
I'm available and willing to assist with code cleanup or any other feature requests that make sense. I write Python regularly at my day job.

**Comment by Chlodochar** [comment]
@rneu31 I don't use Python that often, is #671 the right approach? Could you give it a try?

**Comment by tkSimon** [comment]
Regarding managing the amount of issues and PRs, i just stumbled over this: https://zube.io/
It's free for OSS projects and seems visually appealing as well ;)
